lmn/views/views_notes.py

```diff
@@ -44,3 +55,15 @@ def notes_for_show(request, show_pk):
 def note_detail(request, note_pk):
     note = get_object_or_404(Note, pk=note_pk)
     return render(request, 'lmn/notes/note_detail.html' , { 'note': note })
+
+
+# Function to see if the note already exists
+def note_already_exist(show_pk,user_pk):
+    notes_list = Note.objects.filter(show=show_pk).order_by('-posted_date')
```

Comment:
If you have to do it this way, change the query to search for a note with the show pk and the user pk. `Note.objects.filter(show=show_pk).filter(note=note.pk)`